ner/README.md

```diff
@@ -18,38 +18,79 @@ This assumes you have downloaded the package from the [Cogcomp download page](ht
 
 ### FROM THE COMMAND LINE
 
-Assuming you have plain text files you want to process in directory `input/`, you can generate the annotated versions
-in a new directory (suppose you create `output/`) by navigating to the root directory and running the command:
+To annotate plain text files, navigate to the root directory (`illinois-ner/`), and run the
```

Comment:
Nope. In the downloaded package (at least the one on the website now) the directory is illinois-ner